### DISCUSSION TITLE
Missing Filament profiles

### DISCUSSION THREAD

**Discussion by jeor112** [OrcaSlicer/OrcaSlicer#518] [Q&A] [answered]
Hi
Just installed the OrcaSlicer and it seems, that not all the filament profiles is available in the Prepare Filament dropdown.
I have selected all the profiles for the Bambulab X1C when I installed the slicer.
Have logged out and back in and also restarted the program.

Excample:
The Bambu PETG-CF and Bambu PA-CF isn't available in the dropdown menu.

I have tried to select different bed types, but still the same.

So, is it just me or do I have missed something?

**Comment by jeor112** [comment, marked as answer]
Well, it helps to start a discussion!

I tried to swith from 0.4 nozzle to 0.2 nozzle and back to 0.4 and after that, all the profiles shows up!

Don't know why that happend - I never use 0.2 nozzle...

**Comment by Manfredo1952** [comment]
You could also check the configuration folder (HELP -->  Show Cponfiguration Folder) whether all profiles are correctly copied into the respective folders.  A second start of the program or log out/log in should then shoiw a window like this 
![image](https://user-images.githubusercontent.com/56330568/226153139-5bc051f4-e796-4a03-8e83-4a3643ee6b43.png)

**Reply by jeor112** [reply]
Thanks, yes I did that, but the trick was to swiths between 0.4 to 0.2 and back again.
By second thought, I actually choose all the nozzle sizes when I installed the program, so maybe that's why it had happened.

**Comment by VulpinePrints** [comment]
I just discovered (after wiping my install and all user defined settings) that you can go to the Help menu and select Setup Wizard. This also starts the wizard so you can choose install settings again. Hope this saves a headache somewhere.
